sims.mk

```diff
@@ -6,40 +6,25 @@
 sim-regen-nondeterminism:
 	@echo "Running nondeterminism test..."
 	@go test -mod=readonly $(APP_DIR) -run TestAppStateDeterminism -Enabled=true \
-		-NumBlocks=100 -BlockSize=200 -Commit=true -Period=0 -v -timeout 24h
+		-NumBlocks=50 -BlockSize=100 -Commit=true -Period=0 -v -timeout 24h -tags=experimental
```

Comment:
May be we should have 2 simulations. One for stable as well.

Reply:
Also is there any reason for changing the NumBlocks and BlockSize?

Reply:
Updated to original values. I was testing simulations. `-NumBlocks=100` `-BlockSize=200` taking so much time to finish simulation jobs.